### ISSUE TITLE
Form should not have a border-bottom on the last child

### ISSUE THREAD

**Issue by robbieaverill** [silverstripe/silverstripe-admin#752]
**Version: SS 4.3.0-rc1** (in CWP 2.2.0-rc1 context)

Whenever a form that doesn't have `--no-dividers` or an equivalent class to remove the dividers lines is used, we should exclude the last child from being given a bottom border. Example from asset-admin where there's only one field:

![image](https://user-images.githubusercontent.com/5170590/48345379-8e282c80-e680-11e8-8828-b084a5ef4eb4.png)

**Comment by robbieaverill** [comment]
I've made a PR to fix this particular form factory in place: https://github.com/silverstripe/silverstripe-asset-admin/pull/881

**Comment by ScopeyNZ** [comment]
Is there more form factories that just that one? That PR is now merged.

**Comment by robbieaverill** [comment]
Yeah probably, but I think this issue should be treated separately. If a form doesn't use the `form--no-dividers` class I think we should remove the last child's border
